packages/x-scheduler-headless/src/constants/constants.ts

```diff
@@ -13,3 +13,13 @@ export const SCHEDULER_RECURRING_EDITING_SCOPE: RecurringEventUpdateScope =
 export const EVENT_DRAG_PRECISION_MINUTE = 15;
 
 export const EVENT_DRAG_PRECISION_MS = EVENT_DRAG_PRECISION_MINUTE * 60 * 1000;
+
+/**
+ * Maximum number of days the Agenda view is allowed to scan forward
```

Comment:
I would either add `AGENDA_` in the name or not mention the agenda and has a more generic JSDoc 👍